boilerplate/frontend/angular-prebuilt/src/app/home/home.component.ts

```diff
@@ -1,36 +1,48 @@
-import { AfterViewInit, Component } from "@angular/core";
+import { afterviewinit, component } from "@angular/core";
+import { httpclient } from "@angular/common/http";
 
-import * as Session from "supertokens-web-js/recipe/session";
+import * as session from "supertokens-web-js/recipe/session";
```

Comment:
Id prefer if we kept this and the other things in this file using the existing casing convention.